README.md

```diff
@@ -2,94 +2,31 @@
 
-  * Start a **Terminal**(Mac and Linux) or **Anaconda Prompt**(Windows), then run the following command:
+* Go to https://imjoy.io, connect to the plugin engine. For the first time, you will be asked to fill a token generated by the plugin engine from the previous step. Once connected, you can start to run python plugin through the plugin engine.
 
-  ```
-  conda -V && pip install -U imjoy
-  ```
-
-  * If you encountered any error related to `git` or `pip`, try to run : `conda install -y git pip` before the above command. (Otherwise, please check **FAQs**.)
-
-  To use it after the installation:
-  * Run `imjoy` command in a **Terminal** or **Anaconda Prompt**, and keep the window running.
-
-  * Go to https://imjoy.io, connect to the plugin engine. For the first time, you will be asked to fill a token generated by the plugin engine from the previous step. Once connected, you can start to run python plugin through the plugin engine.
-
-## Upgrading
-
-Normally, the Plugin Engine will upgrade itself when it starts.
-In case you have problem with starting or upgrading the App, try to manually upgrade it by running the following command in a **Terminal**(Mac and Linux) or **Anaconda Prompt**(Windows):
-```
-PATH=~/ImJoyApp/bin:$PATH pip install -U imjoy
-```
 
-## Accessing the ImJoy Engine Conda environment
-If you installed the Plugin Engine with the [ImJoyEngine](https://github.com/oeway/ImJoy-Engine/releases), it will setup an Miniconda environment located in `~/ImJoyApp`.
-
-To access the environment on Linux and Mac, you just need to add `~/ImJoyApp/bin` to your `$PATH`:
-```
-export PATH=~/ImJoyApp/bin:$PATH
-
-# now you can use command such as `imjoy`, `conda`, `pip`, `python` provided from ~/ImJoyApp
-imjoy
-
-```
-For windows, you can use powershell to add the ImJoyApp to `$env.Path`:
-```
-$env:Path = '%systemdrive%%homepath%\ImJoyApp;%systemdrive%%homepath%\ImJoyApp\Library\bin;%systemdrive%%homepath%\ImJoyApp\Scripts;' + $env:Path;
-
-# now you can use command such as `imjoy`, `conda`, `pip`, `python` provided from ~/ImJoyApp
-imjoy
-```
-
-## Uninstall/remove ImJoy Engine
-In order to uninstall or remove ImJoy Engine, you need to remove two folders located in your home/user folder: `ImJoyApp` and `ImJoyWorkspace`.
-
- * `ImJoyApp` contains a Miniconda environemnt and the virtual environemtns used for running ImJoy plugins
- * `ImJoyWorkspace` contains user data for each ImJoy workspace, you may want to backup the data.
+Please notice that if you are trying to use ImJoy Plugin Engine running on a remote server, please use the ImJoy web App served on your server (`http://YOUR_REMOTE_IP:9527`) instead of `https://imjoy.io`. This is because most browser do not allow a web application served throught `https` to connect to a unsecured server (your remote server). Alternatively, you use proxy to enable `https` for the plugin engine, then you will be able to use it with `https://imjoy.io`.
```

Comment:
```suggestion
Please note that if you are trying to use the ImJoy Plugin Engine running on a remote server, please use the ImJoy web App served on your server (`http://YOUR_REMOTE_IP:9527`) instead of `https://imjoy.io`. This is because most browsers do not allow a web application served through `https` to connect to an unsecured server (your remote server). Alternatively, you can use a proxy to enable `https` for the plugin engine. Then you will be able to use it with `https://imjoy.io`.
```